### DISCUSSION TITLE
Bluetooth Disconnect w/ Car Bluetooth

### DISCUSSION THREAD

**Discussion by Lukevos** [NightscoutFoundation/xDrip#2582] [Q&A]
So I updated to Android 13 a while back and ever since then I'll lose Bluetooth connection with my dexcom g6 whenever I connect to my car Bluetooth. As soon as I disconnect from the car it starts working again. What would be causing this? Thanks.

**Comment by Navid200** [comment]
What is the phone brand and model?
Does it happen only with your car?  I mean does it stay connected and working if you connect to another Bluetooth device other than your car?

**Comment by Lukevos** [comment]
I have a Samsung Note 20 ultra on Android 13. It does it on the uconnect headunits in the Jeeps. I've only tried it with two vehicles and they're both Jeeps with uconnect.

**Reply by Navid200** [reply]
And this did not happen with this same phone before updating to Android 13?

**Comment by Lukevos** [comment]
Correct

**Reply by Navid200** [reply]
When it disconnect when you are in your car, what happens if you leave it connected to the car and go into xDrip and tap on restart collector on the classic status page?  Will it not reconnect?

**Reply by Lukevos** [reply]
I don't think it reconnects. I haven't tried that so let me check tomorrow when I drive to work. I did try it with other cars though and it's still the same issue.
